src/lib/server/response.ts

```
@@ -1,4 +1,87 @@
+  message: string;
+  data?: T;
+  errors?: ZodIssue[];
 }
+
+class CustomResponse<T = unknown> extends NextResponse<BaseResponse<T>> {
+  constructor(body?: BodyInit | null, init?: ResponseInit) {
+    super(body, init);
+  }
+
+  static ok<T>(data: T): CustomResponse<T> {
+    return this.json(
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)